src/net/tcp/stream.rs

```diff
@@ -70,12 +70,15 @@ impl TcpStream {
         Self { inner }
     }
 
-    /// Read some data from the stream into the buffer, returning the original buffer and
-    /// quantity of data read.
+    /// Read some data from the stream into the buffer.
+    ///
+    /// Returns the original buffer and quantity of data read.
```

Comment:
No need to change this now, but someday, I would want the order in the comment and the others like it to reflect the order of the values returned. I see the old comment it replaces didn't do that either.

Reply:
Are you referring to the contents of the `BufResult` tuple?   That'd probably be a good idea.